pkg/k8s/k8s.go

```diff
@@ -44,30 +41,18 @@ func generateBaseKubernetesApiUrl(schemeHostAndPort string) (*url.URL, error) {
 	return url, nil
 }
 
-func findK8sConfigFile(override string, contentsOfKubecongigEnvVar string, homeDir string) string {
-	// See https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go
-	if override != "" {
-		return override
+func getConfig(fpath string)(*rest.Config, error) {
```

Comment:
Super tiny nit -- need a space here between the method args and the return args:

```
func getConfig(fpath string) (*rest.Config, error) {
```

If you run gofmt on this file that should fix it.

Reply:
yeah, sorry, forgot to enable gofmt hook in the editor for this project :roll_eyes: should be fixed now